config/v1/types_feature.go

```diff
@@ -67,6 +68,7 @@ type FeatureGateSelection struct {
 	// +optional
 	// +kubebuilder:validation:XValidation:rule="oldSelf == 'CustomNoUpgrade' ? self == 'CustomNoUpgrade' : true",message="CustomNoUpgrade may not be changed"
 	// +kubebuilder:validation:XValidation:rule="oldSelf == 'TechPreviewNoUpgrade' ? self == 'TechPreviewNoUpgrade' : true",message="TechPreviewNoUpgrade may not be changed"
+	// +kubebuilder:validation:XValidation:rule="oldSelf == 'DevPreviewNoUpgrade' ? self == 'DevPreviewNoUpgrade' : true",message="DevPreviewNoUpgrade may not be changed"
```

Comment:
I don't remember there being CEL here, but, LGTM. Is it new?

Reply:
> I don't remember there being CEL here, but, LGTM. Is it new?

"while you were out" :)